### PR TITLE
Create mobile toolbar and landing page for portfolio project

### PR DESCRIPTION
**Pull request by eamador1**
For this project, the following was done:

- [ ] Create a new branch
- [ ] Make a nav-bar following template
- [ ] Make the landing section of the mobile version
- [ ] Install and run linters
- [ ] Create a readme section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,131 @@
...
\ No newline at end of file
+<a name="readme-top"></a>
+# portfolio-project
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```
Comment (stanleySimeon) [inline README.md:9]: Kindly, I suggest you remove the live demo section if your project does not have deployed yet 😄

File: style.css [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  padding: 0;
+  scroll-behavior: smooth;
+}
+
+/*
+font-family: 'Crete Round', serif;
+font-family: 'Inter', sans-serif;
+font-family: 'Poppins', sans-serif;
+font-family: 'Roboto', sans-serif;
+*/
```
Comment (stanleySimeon) [inline style.css:16]: Kindly I suggest you remove all commented code in your code base to make your project look more professional and for best practices reasons. 😄

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="container-header">
+    <nav class="nav-bar">
+      <a class="logo">My Logo</a>
+      <a class="hamburger"><img src="img/hamburger.png" class="hamburger-image" alt="Hamburger Menu"></a>
```
Comment (stanleySimeon) [inline index.html:14]: I noticed that the `<a>` tag in the code you shared is missing the `href` attribute. The href attribute is important for specifying the destination URL that the link should navigate to when clicked. It's essential to include this attribute to ensure proper functionality and accessibility.

Please make sure to add the `href` attribute to the `<a>` tag and provide a valid URL that corresponds to the link's destination. 😄

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @eamador1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Good PR description
- Meaningful commit messages
- Linters check have passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eamador1** [comment]
Since the last review, the following changes were made:

- [ ] Remove live demo section
- [ ] Delete commented code
- [ ] Change <a> tag in lines 13 and 14 to follow right conventions

File: index.html (hunk 3 above)
Comment (eamador1) [reply index.html]: @stanleySimeon I do not see this as a straight forward suggestion, but I get the point😀

**Review by souad988 — APPROVED** [review]
Hi @eamador1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Well implemented design
Good PR description
Good commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**